### ISSUE TITLE
About Rewards

### ISSUE THREAD

**Issue by WendellThompson1996** [warp-zona/solana-trading-bot-v3#4]
I have to say this is a gem of a program and it has been very profitable for me. I just don't have a lot of money and I set harsh trading conditions so I don't know if I can get a bonus?

**Comment by warp-zona** [comment]
> I have to say this is a gem of a program and it has been very profitable for me. I just don't have a lot of money and I set harsh trading conditions so I don't know if I can get a bonus?

Yes sir, you still have plenty of time to reach the bonus threshold by trading, feel free to contact Discord when you reach it.

Also, please be aware of the market risk, your money is more important than the bonus.
